packages/teleport/src/Recordings/RecordingsList.tsx

```diff
@@ -81,8 +95,18 @@ export default function RecordingsList(props: Props) {
   );
 }
 
+const renderIconCell = (type: RecordingType) => {
+  const icon = type === 'ssh' ? Icons.Cli : Icons.Desktop;
+
+  return (
+    <Cell>
+      <Icon p={1} mr={3} fontSize={3} as={icon} />
+    </Cell>
+  );
+};
+
 const renderPlayCell = ({ description, sid }: Recording, clusterId: string) => {
-  if (description !== 'play') {
+  if (description !== PlayDescription) {
```

Comment:
sorry this does not seem quite right when description text is used as a condition. What if we add a new property `playable` and use it instead? 

Plus, remove this type `PlayDescription` and use a regular `const` instead (if the same description text is used in several places)

Reply:
Sure. Since we're here, what do you think about just removing non-playable sessions from the table ([ref](https://github.com/gravitational/teleport/pull/9839#issuecomment-1016767764))?

Reply:
that was product/ux decision